lib/fluent/plugin/out_aws-elasticsearch-service.rb

```diff
@@ -17,7 +17,6 @@ class AwsElasticsearchServiceOutput < ElasticsearchOutput
       config_param :access_key_id, :string, :default => ""
       config_param :secret_access_key, :string, :default => "", secret: true
       config_param :assume_role_arn, :string, :default => nil
-      config_param :ecs_container_credentials_relative_uri, :string, :default => nil #Set with AWS_CONTAINER_CREDENTIALS_RELATIVE_URI environment variable value
```

Comment:
Removing config_param is not good.
If you remove it completely, please use [`:obsolete`](https://github.com/fluent/fluentd/blob/master/lib/fluent/config/section.rb#L197) here.

Reply:
It's certainly not good.
Modify.

Reply:
How about refer `AWS_CONTAINER_CREDENTIALS_RELATIVE_URI` in `#configure`?

```ruby
aws_container_credentials_relative_uri = opts[:ecs_container_credentials_relative_uri] || ENV["AWS_CONTAINER_CREDENTIALS_RELATIVE_URI"]
```

Reply:
Ah, this should be in `#credentails`.
But, using instance variable to store credentials information in `#configure` is also OK for me